specs/phase1/beacon-chain.md

```diff
@@ -695,7 +696,7 @@ def process_operations(state: BeaconState, body: BeaconBlockBody) -> None:
     # See custody game spec.
     process_custody_game_operations(state, body)
 
-    process_crosslinks(state, body.shard_transitions, body.attestations)
+    process_shard_transitions(state, body.shard_transitions, body.attestations)
```

Comment:
Would be neat to convert to: 
`for_ops(body.shard_transitions, body.attestations, process_shard_transitions)`
But it doesn't look feasible today 😅

Reply:
But `for_ops` is mainly for executing operation one-by-one, and we need all `body.shard_transitions` and `body.attestations` to process crosslinks.

Reply:
Makes sense!